README.md

```diff
@@ -72,6 +72,32 @@ In[] := HypergraphPlot[
 
 Exploring the models of this more complicated variety is what this package is mostly designed for.
 
+# Getting Started
+
+## Prerequisites
+
+To start using `SetReplace` you only need two things.
+
+* [Wolfram Language 12.1+](https://www.wolfram.com/language/) including [WolframScript](https://www.wolfram.com/wolframscript/). A free (although not open-source) version is available as [Wolfram Engine](https://www.wolfram.com/engine/).
```

Comment:
I think we should say "A free version is available as Wolfram Engine". Not no-cost

Reply:
Ok, I agree. I was trying to avoid confusion with free as open-source, but I guess it's clear from the context.